### ISSUE TITLE
Bugged character after touch an Me Anihilation Plate

### ISSUE THREAD

**Issue by Enderlook** [AppliedEnergistics/Applied-Energistics-2#3181]
- Minecraft Version: 1.12.2
- AE2 Version: appliedenergistics2-rv5-beta-2
- Forge Version: forge-1.12.2-14.23.0.2512

I was just trying to make an automated fluix crystal crafter when I build an ME Annihilation Plane and I did right click to it in order to check if it has a configurable GUI (there is some time I don't use this mod so I didn't remember if it had a GUI) and in that moment the block (with its cable) dissapear and I fall 1 block heigh (I was above the block) and after that moment the character get quite buggy.
My character wasn't able to break blocks or even change of item from the hotbar (press 1-9 or use the mouse wheel). I restart the game and that didn't fix the bug so I killed myself... that fixed the bug.

Sorry but I don't have any crash report because the game didn't crash.

PS: The 2 block (Plane and cable) dissapeared.

**Comment by fscan** [comment]
Any log file would be helpful. You can force a crash report pressing F3+c.
I have no idea where to start with this one, never heard of an issue like this.

**Comment by Enderlook** [comment]
Oh, so sorry fscan. I have just reproduced the bug to get a crash report when I actually found what produced it.
It isn't your mod, I also use "CarryOn" that let me carry chest if I press Shift + Right Click and while I carry a chest I can't break blocks. Futhemore chests this mod lets carry another blocks of mods that think their could have GUI or inner storage, I accidentally must press shift while I was trying to check if you block has a GUI and the mod misunderstand the action and took the block.
I'm sorry for have annoy you.
PS: I don't have any idea of how to delete this thread (issue).
